haskell/test/Test/SegCFMM/Contract.hs

```diff
@@ -14,4 +14,4 @@ import Util (fetchContract)
 
 segCFMMContract :: Contract (ToT Parameter) (ToT Storage)
 segCFMMContract =
- $(fetchContract @(ToT Parameter) @(ToT Storage) "SEGMENTED_CFMM_PATH")
+ $(fetchContract @Parameter @Storage "SEGMENTED_CFMM_PATH")
```

Comment:
Btw, we could start using `embedContractM` available in morley now instead of `fetchContract`.

Then `originateSegCFMM` can also be rewritten just in one line if `originateSimple` is used.